sdks/python/apache_beam/runners/portability/universal_local_runner.py

```diff
@@ -72,6 +74,9 @@ def __init__(
     self._job_service_lock = threading.Lock()
     self._subprocess = None
     self._runner_api_address = runner_api_address
+    self._docker_image = (
+      docker_image
+      or os.environ['USER'] + '-docker.apache.bintray.io/beam/python:latest')
```

Comment:
Can we document the docker default image name assumption and print a log about telling which docker image is in use.

Reply:
I think this is built by the gradle task, right @bsidhom ?

In any case, I've put this into a method and added logging to make things more clear.

Reply:
@robertwb That's correct. The [python container build task](https://github.com/apache/beam/blob/a20ba182e3b4050f25b2f2cf72dda06443d76afb/sdks/python/container/build.gradle#L49) [defaults to](https://github.com/apache/beam/blob/a20ba182e3b4050f25b2f2cf72dda06443d76afb/build_rules.gradle#L1078) `$USER-docker-apache.bintray.io/beam/$SDK:latest`.